packages/terra-functional-testing/src/commands/expect/toBeAccessible.js

```diff
@@ -0,0 +1,19 @@
+const { runAxe } = require('../axe');
+
+/**
+ * Invokes axe analysis on the current document and asserts there are no accessibility violations.
+ * @param {*} _ - Unused param. The expect is always run on the current document.
+ * @param {Object} options - Optional axe configuration overrides.
+ * @returns {Object} - An object that indicates if the assertion passed or failed with a message.
+ */
+function toBeAccessible(_, options = {}) {
+  const { result } = runAxe(options);
+  const { violations } = result;
```

Comment:
Do we need to pull forward this disabled rule? 

https://github.com/cerner/terra-toolkit-boneyard/blob/main/src/wdio/services/TerraCommands/chai-methods.js#L11

Reply:
Yes, I think we should disable it in a slightly different way though. I think it should be disabled prior to running axe, rather than filtering it out of the returned violations. I'll work on adding it into this PR

Reply:
Disabled the rule prior to calling axe, rather than filtering it out of the violation array:

https://github.com/cerner/terra-toolkit/pull/452/commits/1aad2e2c77dc3a59fed32b3ce8c763a95b57d952